src/main/java/org/codehaus/plexus/archiver/zip/ZipResource.java

```diff
@@ -45,14 +45,15 @@ private static long getLastModofied( ZipArchiveEntry entry )
 
     public synchronized PlexusIoResourceAttributes getAttributes()
     {
-        int mode = entry.getUnixMode();
-        if ( ( mode & UnixStat.FILE_FLAG ) == UnixStat.FILE_FLAG )
+        int mode = -1;
```

Comment:
I think it's better to use constant. But didn't found existing one? Maybe we can introduce new one in `PlexusIoResourceAttributes`. For example `UNKNOWN_OCTAL_MODE`